sys/vane/rver.hoon

```diff
@@ -627,9 +627,7 @@
     }
   };
 
-  export function newChannel() {
-    return new Channel;
-  }
+  window.urb = new Channel();
```

Comment:
My understanding is that this makes a single channel object on window, instead of giving the caller the option of how this library gets used. If the ES6 syntax for this isn't cross-browser compatible, is there a way to make this at least export newChannel() to callers and have callers use it instead of having a library modify a global object?

Reply:
In the current JS, Channel becomes a globally available type that any application can instantiate.
The ES6 syntax was only supported in Chrome. I see a good argument for not instantiating `window.urb = new Channel();` however we do not need to export the function newChannel(), as the Channel class will already be available.

Reply:
OK, then just remove the assignment here and use Channel where needed.